app/controllers/bgeigie_imports_controller.rb

```diff
@@ -34,6 +34,13 @@ def approve
     redirect_to @bgeigie_import
   end
 
+  def fixdrive
+    @bgeigie_import = BgeigieImport.find(params[:id])
```

Comment:
looks like a potential attack factor as a scope is not used, same in other places?